thumbor/url_composer.py

```diff
@@ -220,6 +220,8 @@ def main(arguments=None):
     """Converts a given url with the specified arguments."""
 
     parsed_options, arguments = get_options(arguments)
+    if not arguments:
+        sys.exit(1)
```

Comment:
Maybe print a message saying arguments are required?

Reply:
There is a message in [get_options()](https://github.com/thumbor/thumbor/blob/master/thumbor/url_composer.py#L163), I will move it to `main()`
```
Error: The image argument is mandatory. For more information type thumbor-url -h
```